group_vars/all/louketo.yml

```diff
@@ -0,0 +1,12 @@
+---
+louketo_instances:
+  - name: freescout
```

Comment:
Could this be seperated into a template?

Reply:
It could, but this would be more effort to maintain. I think it is much easier when it is in the variables, since this is also just yaml